index.js

```diff
@@ -60,15 +60,63 @@ renderPlayer(player)
 
 
 /***** Deliverable 1 *****/
-function toggleColor(element) {
-  if (element.style.color === "red") {
-    element.style.color = "black"
+// function toggleColor(element) {
+//   if (element.style.color === "red") {
+//     element.style.color = "black"
+//   } else {
+//     element.style.color = "red"
+//   }
+// }
+let header = document.querySelector("h1#header")
+
+header.addEventListener("click", function(){
+  if (header.style.color === "red") {
+    header.style.color = "black"
   } else {
-    element.style.color = "red"
+    header.style.color = "red"
   }
-}
+})
```

Comment:
This works, but you can also solve it by using the original `toggleColor` function like this:

```js
function toggleColor(element) {
  if (element.style.color === "red") {
    element.style.color = "black"
  } else {
    element.style.color = "red"
  }
}

const header = document.querySelector("#header")
header.addEventListener("click", function(event) {
  toggleColor(header)
})
```